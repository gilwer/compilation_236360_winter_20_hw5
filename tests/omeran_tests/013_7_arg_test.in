void foo(int a, int bb, int c, int d, int e, int f) {
    print("six parameters for foo");
    printi(a+bb+c+d+e+f);
    return;
}

void bar(int a, int bb, int c, int d, int e, int f, int g) {
            print("seven parameters for bar");
            printi(a+bb+c+d+e+f+g);
            return;
        }

void main() {
    foo(1,2,3,4,5,6);
    bar(1,2,3,4,5,6,7);
}