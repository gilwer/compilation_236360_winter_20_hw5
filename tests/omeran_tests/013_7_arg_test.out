six parameters for foo
21
seven parameters for bar
28
